spec/exp_descriptor_en_spec.rb

```diff
@@ -311,6 +311,10 @@ def desc(expression, opts = {})
       expect(desc('23 12 * JAN-MAR * 2013-2015')).to eq('At 12:23 PM, January through March, 2013 through 2015')
     end
 
+    it 'hours ranges' do
+      expect(desc("10,11 2,4-5 * * *")).to eq('At 10 and 11 minutes past the hour, between 2:00 AM,4:00 AM and 5:59 AM')
```

Comment:
according to the original design that follows  https://github.com/bradymholt/cron-expression-descriptor, the word `through` is supposed to be used for ranges

like in the following examples:

https://cronexpressiondescriptor.azurewebsites.net/?expression=10%2C11+2%2C4-5+*+*+*&locale=en
```
10,11 2,4-5 * * *
At 10 and 11 minutes past the hour, at 02:00 AM and 04:00 AM through 05:59 AM`
```

https://cronexpressiondescriptor.azurewebsites.net/?expression=10%2C11+2%2C4-5%2C7-9+*+*+*&locale=en
```
10,11 2,4-5,7-9 * * *
At 10 and 11 minutes past the hour, at 02:00 AM, 04:00 AM through 05:59 AM, and 07:00 AM through 09:59 AM
```
also, even if you do use the wording `between` ,in this particular example it should follow after `2:00 AM`, e.g. `at 2:00 AM, between 4:00 AM and 5:59 AM`